docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -167,16 +202,93 @@ \section{Expected Technology}
 
 \begin{itemize}
 \item Specific programming language
+\subitem The project can roughly be divided into 3 main components:
+\begin{enumerate}
+  \item A database where the bulk of the site's data will be stored.
+  \item A webserver which will host the website's visual data and code.
+  \item Middleware which will allow for communication between the 
+  webserver and database.
+\end{enumerate}
+\subitem The middleware will be written in Python due to its ease of use and 
+familiarity among team members.
+\subitem We will either use PostgreSQL scripts or an ORM (like SQLalchemy) 
+to implement the database, the language used will depend on the implementation 
+chosen. PostgreSQL scripts would require using only SQL, while an ORM would 
+likely live where with middleware and as such would be written in Python.
+\subitem The webserver will be written in Javascript using the React framework.
+This is because of its ease of use, team member experience, and large set 
+of available libraries which will be useful for implementing the visual elements
+of the website. Additionally, it was expressed to us by stakeholders that the 
+website should be easily maintainable. Implementing the website using an 
+extremely popular and widespread framework such as React means that there 
+are countless resources online to help future maintainers keep the project alive.
+
 \item Specific libraries
+\subitem Depending on the implementation, the middleware will use some combination
+of the FastAPI and SQLalchemy libraries. FastAPI will be used to implement HTTPS 
+commmunication rotuers between the database and webserver. SQLalchemy will be used to 
+implement the database if it is decided that it will be implemented using an ORM.
+\subitem The webserver will use a large set of both funcational and visual react
+libraries. For example react-navigation will be used for its page traversal 
+functionality while libraries such as react-datepicker and react-calendar will 
+be very helpful in implement the visual elements of a scheduling system. 
```

Comment:
should be "implementing" instead of "implement"
Also above put a comma after "For example,"